config/rbac/role.yaml

```diff
@@ -41,13 +41,13 @@ rules:
   - update
   - watch
 - apiGroups:
-  - release.giantswarm.io.giantswarm.io
+  - release.giantswarm.io
   resources:
   - releases/finalizers
```

Comment:
The finalizers are not a subresource, right? I don't think you need RBAC for that field.